libpdbg/ocmb.c

```diff
@@ -37,34 +40,94 @@ static struct sbefifo *ocmb_to_sbefifo(struct ocmb *ocmb)
 	return sbefifo;
 }
 
+static struct sbefifo *ody_ocmb_to_sbefifo(struct ocmb *ocmb)
+{
+	// ocmb targets from system device tree have proc and index
+	// sbefifo backend targets from bmc backend device tree has
+	// proc, ocmb index and device path
+	// Perform map from system device tree to bmc-backend device tree
+	// for ocmb target using proc and index to get device path.
+	uint32_t ocmb_proc = pdbg_target_index(pdbg_target_parent("proc", 
+							&ocmb->target));
```

Comment:
What is the reason we are not doing a `% 0x8` like we did at LN52? Interested to know